modules/caddyhttp/fileserver/browse.go

```diff
@@ -48,7 +48,8 @@ var BrowseTemplate string
 // Browse configures directory browsing.
 type Browse struct {
 	// Filename of the template to use instead of the embedded browse template.
-	TemplateFile string `json:"template_file,omitempty"`
+	TemplateFile  string `json:"template_file,omitempty"`
+	RevealSymLink bool   `json:"reveal_symlink,omitempty"`
```

Comment:
Please add a godoc comment; we parse godoc comments to generate our docs https://caddyserver.com/docs/json/apps/http/servers/routes/handle/file_server/browse/

Reply:
This should now be good to go.